app/components/admin/custom_fields/hierarchy/item_component.rb

```diff
@@ -35,31 +35,33 @@ class ItemComponent < ApplicationComponent
         include OpTurbo::Streamable
         include OpPrimer::ComponentHelpers
 
-        def initialize(custom_field:, hierarchy_item:, edit_item_form_data: { show: false })
-          super
-          @custom_field = custom_field
-          @hierarchy_item = hierarchy_item
+        def initialize(item:, edit_item_form_data: { show: false })
```

Comment:
I am thinking we could also change the `edit_item_form_data` parameter to be just a boolean, since we don't pass down other attributes along anymore.

e.g.
```ruby
def initialize(item:, show_edit_form: false)
```

Possibly we can do that at a separate moment, though